modules/ixBidAdapter.md

```diff
@@ -146,18 +179,92 @@ var adUnits = [{
     }]
 }];
 ```
-
+**Video (Instream):**
+```javascript
+var adUnits = [{
```

Comment:
Hi,

I am not able to get a valid bid response when using this example ad unit.

Can you please update or let me know hoe to solicit a valid video ad so I can test out the changes a little more thoroughly?

Reply:
Hello,
Thanks for looking it over!
Here's an example config: 
```
var videoAdUnit = {
    code: '/21793097377/el-testo-prebid-video-vast-a',
    mediaTypes: {
        video: {
            playerSize: [
                [640, 480]
            ],
            context: 'instream'
        }
    },
    bids: [{
        bidder: 'ix',
        params: {
            siteId: '335943',
            size: [640, 480],
            video: {
                protocols: [2, 3, 5, 6],
                mimes: [
                    'video/mp4',
                    'video/webm',
                    'video/3gpp',
                    'video/x-mpegURL',
                    'application/javascript',
                    'application/x-shockwave-flash'
                ],
                minduration: 0,
                maxduration: 60,
                linearity: 1,
                api: [1, 2],
                playbackmethod: [3],
                startdelay: 0
            }
        }
    }]
};
```
We wouldn't want to return bids from examples in our doc, though, as we'd really prefer the touch point before a partner goes live.

This is largely resubmitting this PR from earlier this year: https://github.com/prebid/Prebid.js/pull/3901

Please let me know if there are further questions.

Regards,
Dmitry
Index Exchange